tutorials/epochs/plot_20_visualize_epochs.py

```diff
@@ -131,6 +131,32 @@
 epochs['auditory'].plot_psd(picks='eeg')
 
 ###############################################################################
+# It is also possible to plot spectral estimates across sensors as a scalp
+# topography, using :meth:`~mne.Epochs.plot_psd_topomap`. The default
+# parameters will plot five frequency bands (δ, θ, α, β, γ), and will plot
+# the power estimates in decibels:
+
+epochs['visual/right'].plot_psd_topomap()
```

Comment:
I think we should also give the intervals used to define the bands in the topo subtitles.
There is perfect consensus on what are the "correct" bands (for good reasons). It's just a good default but it should be explicit.

Reply:
also what do you think about showing units on the colorbar following the way it's done with plot_psd?